README.md

```diff
@@ -56,6 +56,26 @@ Most of Sidekiq's overhead is Redis network I/O.
 ActiveJob adds a notable amount of CPU overhead due to argument deserialization and callbacks.
 Concurrency of 30 was determined experimentally to maximize one CPU without saturating it.
 
+Performance Dragonfly
+-------------------
+
+The benchmark above was slightly revamped in `bin/multi_queue_bench` to scale better with multi-core systems like Dragonfly.
+We used a c5.4xlarge to run Dragonfly and a monstrous 96 core c5a.24xlarge to run the benchmark. All job types are no-op
+Sidekiq::Job. We run Dragonfly with `dragonfly --proactor_threads=n --shard_round_robin_prefix="queue"`. `n` is the number 
+of threads that Dragonfly uses and it defaults to the number of threads available on the system. This is different than Redis, 
+which is single threaded and scales vertically via the Redis cluster. This benchmark is all about scale, and therefore we
+run it with 3 different setups. `RUBY_YJIT_ENABLE=1 PROCESSES=96 QUEUES=k THREADS=10 ./multi_queue_bench` with `k` 1, 2, 8.
+We used Ruby with version 3.0.2p107 and Sidekiq trunk.
```

Comment:
Ruby 3.0 is quite old and almost EOL. I would urge you to redo the benchmarks with Ruby 3.3.

Reply:
I rerun the benchmarks with 3.3.0, roughly same performance :)

Reply:
Then please update the text.